app/views/qsets/show.html.erb

```diff
@@ -24,50 +24,58 @@
         <div class="panel-heading">
           <h4 class="panel-title">
             <div class="row valign">
-              <div class="col-sm-1">
-                <div class="net-votes"><p id="question-<%= question.id %>-net-votes"><%= question.plusminus %></p><br></div>
-              </div>
-              <div class="col-sm-1">
-                <div class="question-voting">
-                  <a class="upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>">Up</a><br>
-                  <a class="downvote-button" data-vote="downvote" data-vote-qid="<%= question.id %>">Down</a>
+              <div class="col-xs-9 col-sm-9">
```

Comment:
Bootstrap doesn't require identical column width declarations at multiple levels, we can just use the smallest one.